Code/Framework/AzCore/Tests/Debug/PerformanceCollectorTests.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/UnitTest/UnitTest.h>
+#include <AzTest/AzTest.h>
+
+#include <AzCore/Debug/PerformanceCollector.h>
+#include <AzCore/JSON/document.h>
+
+using namespace AZ;
+using namespace Debug;
+
+namespace UnitTest
+{
+    class PerformanceCollectorTest
+        : public AllocatorsFixture
+    {
+    public:
+        PerformanceCollectorTest()
+        {
+        }
+
+        void SetUp() override
+        {
+            AllocatorsFixture::SetUp();
+        }
+
+        ~PerformanceCollectorTest() override
+        {
+        }
+
+        void TearDown() override
+        {
+            AllocatorsFixture::TearDown();
+        }
```

Comment:
ScopedAllocatorSetupFixture should be used instead.
```suggestion
    class PerformanceCollectorTest
        : public ScopedAllocatorFixture
    {
    };
```

Reply:
Good to know, but will pass on it. As during UnitTests I really don't want to create files in jenkins. Left this comment: